src/stream.cc

```diff
@@ -24,140 +24,110 @@
 
 namespace wabt {
 
-static FileStream s_stdout_stream;
-static FileStream s_stderr_stream;
-
-void init_stream(Stream* stream, Writer* writer, Stream* log_stream) {
-  stream->writer = writer;
-  stream->offset = 0;
-  stream->result = Result::Ok;
-  stream->log_stream = log_stream;
+Stream::Stream(Writer* writer, Stream* log_stream)
+    : writer_(writer),
```

Comment:
Yeah, I don't really care either way. But it is better than m_foo. And I like making a distinction between private members and local variables.